api/src/integration/java/org/pmiops/workbench/NotebooksIntegrationTest.java

```diff
@@ -3,47 +3,23 @@
 import static com.google.common.truth.Truth.assertThat;
 
 import org.junit.Test;
-import org.mockito.Mock;
-import org.pmiops.workbench.config.WorkbenchConfig;
 import org.pmiops.workbench.notebooks.LeonardoNotebooksClient;
 import org.pmiops.workbench.notebooks.LeonardoNotebooksClientImpl;
-import org.pmiops.workbench.notebooks.NotebooksRetryHandler;
-import org.pmiops.workbench.notebooks.api.ClusterApi;
-import org.pmiops.workbench.notebooks.api.NotebooksApi;
-import org.pmiops.workbench.test.Providers;
-import org.pmiops.workbench.workspaces.WorkspaceService;
-import org.springframework.retry.backoff.NoBackOffPolicy;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Import;
 
-/** Created by brubenst on 5/8/18. */
-public class NotebooksIntegrationTest {
-  /*
-   * Mocked service providers are (currently) not available for functional integration tests.
-   * Integration tests with real projects/clusters against production Notebooks is not
-   * recommended at this time.
-   */
+public class NotebooksIntegrationTest extends BaseIntegrationTest {
+  @Autowired private LeonardoNotebooksClient leonardoNotebooksClient;
 
-  @Mock private ClusterApi clusterApi;
-  @Mock private NotebooksApi notebooksApi;
-  @Mock private WorkspaceService workspaceService;
-
-  private final LeonardoNotebooksClient leonardoNotebooksClient =
-      new LeonardoNotebooksClientImpl(
-          Providers.of(clusterApi),
-          Providers.of(notebooksApi),
-          Providers.of(createConfig()),
-          Providers.of(null),
-          new NotebooksRetryHandler(new NoBackOffPolicy()),
-          workspaceService);
+  @TestConfiguration
+  @ComponentScan("org.pmiops.workbench.notebooks")
+  @Import({LeonardoNotebooksClientImpl.class})
+  static class Configuration {}
 
   @Test
   public void testStatus() {
     assertThat(leonardoNotebooksClient.getNotebooksStatus()).isTrue();
```

Comment:
I'd rename `getNotebooksStatus` to `isNotebooksServiceUp()`, as it better matches what it does and corresponds to a boolean.

Again, I'd rename this to `NotebooksSmokeTest`. I expected a lot more actual testing considering the boilerplate.